project/Paradox.scala

```diff
@@ -34,6 +34,10 @@ object Paradox {
         "javadoc.akka.base_url" -> "https://doc.akka.io/japi/akka/2.6",
         "javadoc.akka.link_style" -> "direct",
         "javadoc.akka.http.base_url" -> "https://doc.akka.io/japi/akka-http/current",
+        "javadoc.com.fasterxml.jackson.annotation.base_url" -> "https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-annotations/latest/",
+        "javadoc.com.fasterxml.jackson.annotation.link_style" -> "direct",
+        "javadoc.com.fasterxml.jackson.databind.base_url" -> "https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-databind/latest/",
+        "javadoc.com.fasterxml.jackson.databind.link_style" -> "direct",
         "javadoc.akka.http.link_style" -> "frames",
```

Comment:
```suggestion
        "javadoc.akka.http.base_url" -> "https://doc.akka.io/japi/akka-http/current",
        "javadoc.akka.http.link_style" -> "frames",
        "javadoc.com.fasterxml.jackson.annotation.base_url" -> "https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-annotations/latest/",
        "javadoc.com.fasterxml.jackson.annotation.link_style" -> "direct",
        "javadoc.com.fasterxml.jackson.databind.base_url" -> "https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-databind/latest/",
        "javadoc.com.fasterxml.jackson.databind.link_style" -> "direct",
```
Keep the `akka.http` properties together.